src/widgets/current-refined-values/current-refined-values.js

```diff
@@ -233,7 +233,7 @@ function clearRefinementFromState(state, refinement) {
   case 'exclude':
     return state.removeExcludeRefinement(refinement.attributeName, refinement.name);
   case 'numeric':
-    return state.removeNumericRefinement(refinement.attributeName, refinement.operator, refinement.name);
```

Comment:
I do prefer fixing the issue at the root level than parsing a string as an int that was previously an int.